build-container-locally.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
-
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
```

Comment:
Might be better to add a space here, in case we need to run the tool a second time ? If we did, this comment it's joined onto will be wiped-out ?